backend/app/infrastructure/database/models.py

```diff
@@ -174,3 +174,8 @@ class MultisigSignatures(BaseModel):
     status = Column(db.String, nullable=False)
     user_ip = Column(db.String, nullable=False)
     confirmed_signature = Column(db.String, nullable=True)
+
+
+class ScoreDelegation(BaseModel):
+    id = Column(db.Integer, primary_key=True)
+    hashed_addr = Column(db.String, nullable=False)
```

Comment:
(future performance consideration) i'm wondering if we shouldn't put additional ENUM-alike field in the table, i.e FIRST, SECONDARY, BOTH to distinguish which hash we look for in the be's functions; thanks to that we could filter only specific types of records without the need of iterating over everything every time

Reply:
I'm wondering if it wouldn't make it less secure. As long as we don't really need it I think we can stick to this approach

Reply:
kk